libraries/freertos_plus/standard/freertos_plus_tcp/include/FreeRTOS_Sockets.h

```diff
@@ -345,10 +345,17 @@ typedef union xTCP_UDP_HANDLER
 	FOnUDPSent_t	pxOnUDPSent;		/* FREERTOS_SO_UDP_SENT_HANDLER */
 } F_TCP_UDP_Handler_t;
 
-BaseType_t FreeRTOS_setsockopt( Socket_t xSocket, int32_t lLevel, int32_t lOptionName, const void *pvOptionValue, size_t xOptionLength );
+BaseType_t FreeRTOS_setsockopt( Socket_t xSocket, int32_t lLevel, int32_t lOptionName, const void *pvOptionValue, size_t uxOptionLength );
 BaseType_t FreeRTOS_closesocket( Socket_t xSocket );
-uint32_t FreeRTOS_gethostbyname( const char *pcHostName );
```

Comment:
Removal of this is causing errors in `libraries\abstractions\secure_sockets\freertos_plus_tcp\iot_secure_sockets.c(211)`. May need to remove the call from there as well.

Reply:
I found that `FreeRTOS_gethostbyname()` was added to `FreeRTOS_Sockets.h`, whereas it should be declared in `FreeRTOS_DNS.h` only.
When changing the code I didn't realise that the secure socket modules do not include `FreeRTOS_DNS.h`.

It would be most correct to include `FreeRTOS_DNS.h`, but that might have side-effects.
I will put the declaration back in `FreeRTOS_Sockets.h` for now.

Reply:
Thanks